src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/lib/validate_headers.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+
+export default function validateHeaders(fields) {
+  if (!_.isArray(fields)) {
+    throw new Error('first argument must be an array');
+  }
+
+  const errors = [];
+
+  const fieldsWithPositions = _.reduce(fields, (result, field, index) => {
+
+    (result[field] || (result[field] = [])).push(index + 1);
+    return result;
+  }, {});
+
+  _.forEach(fieldsWithPositions, function (positions, field) {
+
+    // empty field errors
+    if (_.isEmpty(field)) {
+      errors.push({
+        'positions': positions,
+        'type': 'blank'
+      });
+    }
+
+    // duplicate field errors
+    else if (positions.length > 1) {
+      errors.push({
+        'fieldName': field,
+        'positions': positions,
+        'type': 'duplicate'
```

Comment:
These object literal keys don't need quotes. I noticed there were some of these in the test file as well.